src/components/tasks/card/ProgressText.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import classNames from '@/components/tasks/card/card.module.scss';
+import { handleProgressTextProps } from '@/interfaces/task.type';
+
+const HandleProgressText: FC<handleProgressTextProps> = ({
+    progress,
+    handleSaveProgressUpdate,
+    handleProgressUpdate,
+}) => {
+    if (progress) {
+        return (
+            <div
+                className={classNames.changeProgressText}
+                onClick={() => handleSaveProgressUpdate()}
+            >
+                save Progress
+            </div>
+        );
+    } else {
```

Comment:
- Should we do a default return instead of using else?
The code will be less nested.

Reply:
fixed